### DISCUSSION TITLE
A host in subnets can't have more than one IP address

### DISCUSSION THREAD

**Discussion by lijh8** [strongswan/strongswan#537] [General]
Hi experts,

```
    172.168.10.0/24 -- | 192.168.0.1 | === | 192.168.0.2 | -- 172.168.30.0/24
       moon-net             moon                sun              sun-net
```

I assigned two IP addresses for a CentOS 7.3 host in left side moon-net.
The host in right side sun-net has one IP address.

The tunnel has been setup. I can see traffic with ESP in tcpdump.

From left side host, I can ping host in right side.
From right side host, I can't ping host in left side. The left host with two IP addresses won't send the reply package. Why is that? I don't know much about networking. Could you please teach me.

Thanks

IP addresses for the host in left moon-net:

```
#192.
IPADDR0=192.168.10.160
NETMASK0=255.255.255.0
GATEWAY0=192.168.10.1

#172.
IPADDR1=172.168.10.160
NETMASK1=255.255.255.0
GATEWAY1=172.168.10.1
```

**Comment by Thermi** [comment]
Please provide all information as listed in #196.
